earth_enterprise/tutorial/GoogleEarthFusionTutorial.spec

```diff
@@ -1,49 +0,0 @@
-%define _prefix /opt/google/share/tutorials/fusion
```

Comment:
I would suggest to keep files related to the build system and rpm in repo (.rpmrc, GoogleEarthFusionTutorial.spec, SConstruct, Makefile), so they can be easily edited/updated, and only put FusionTutorial folder (data files) into the tarball and on geeoss-site.

Agreed, would be nice to add README.md with the link to download FusionTutorial.
Thanks.